pkg/pods/testdata/taskrunspec-typo.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2020 The Tekton Authors.
```

Comment:
this should be 2023

Reply:
fixed @ https://github.com/tektoncd/cli/pull/1969/commits/5f4ffd50bf7d7295467e2a5d3828ba2f2cfc3c4b